### DISCUSSION TITLE
Pressure fluctuations

### DISCUSSION THREAD

**Discussion by SaiSantosh1824** [firemodels/fds#13404] [General]
Hi, 
I have been simulating a number of corridor ventilation scenarios (say around 20). Of them 3 of the scenarios exhibited random pressure fluctuation (see attached video as an example). As the methodology for modelling across the scenarios have not changed I am confused why this fluctuation is occurring on select few cases. I am also not able to pinpoint the reason for this fluctuation and would appreciate some help on this.

I have also attached the FDS file for that scenario.

[Pressure.zip](https://github.com/user-attachments/files/16909305/Pressure.zip)
[Test_Case.txt](https://github.com/user-attachments/files/16909340/Test_Case.txt)

Thank you!

**Comment by drjfloyd** [comment]
Generated by PyroSim - Version 2019.3.1204

Are you using the version of FDS for this version of Pyrosim (a five year old FDS) or the current FDS?  If you are not using the current FDS, try that. The HVAC pressure solution is not directly coupled with the pressure solution in the meshes. We keep refining how we deal with coupling HVAC to FDS to try and limit these kinds of oscillations.

**Comment by SaiSantosh1824** [comment]
Hi drjfloyd,

I have been using FDS 6.7.1 which I believe is the older version of the FDS as opposed to the latest version 6.9.1. I will set this sim up using the latest version and check whether the same pressure fluctuation is observed and update.

It is however strange that only a few cases are affected by this and just out of curiosity, is it possible to indicate which element in the sim might be causing this issue or find a solution for this on older version of FDS?

**Reply by drjfloyd** [reply]
It may not be a specific element but more how all the elements interact given the specific conditions of that simulation.

**Reply by SaiSantosh1824** [reply]
Fair point!
I have set the sim up using the latest version of the FDS. I will update once the sim has completed.

**Reply by SaiSantosh1824** [reply]
Update, there are no pressure oscillations when running the simulation on the latest version!
Guess I will have to update to the latest version going forward 
Thanks for the help on this!
Thank you
